api/src/application/user/user-views.js

```diff
@@ -0,0 +1,229 @@
+            selected: getValue('zoneinfo') === name
+          })),
+          value: getValue('zoneinfo'),
+          error: validationErrorMessages.zoneinfo,
+        },
+        {
+          name: 'locale',
+          label: 'Locale',
+          isDropdown: true,
+          options: Object.keys(userFormData.locales).map((value) => ({
+            label: userFormData.locales[value],
+            value,
+            selected: getValue('locale') === value,
+          })),
+          value: getValue('locale'),
+          error: validationErrorMessages.locale,
+        },
+        {
+          name: 'phone_number',
+          label: 'Phone Number',
+          type: 'text',
+          value: getValue('phone_number'),
+          error: validationErrorMessages.phone_number,
+        },
+        {
+          name: 'address.street_address',
+          label: 'Street Address',
+          type: 'text',
+          value: getValue('address.street_address'),
+          error: validationErrorMessages['address.street_address'],
+        },
+        {
+          name: 'address.locality',
+          label: 'Locality',
+          type: 'text',
+          value: getValue('address.locality'),
+          error: validationErrorMessages['address.locality'],
+        },
+        {
+          name: 'address.region',
+          label: 'Region',
+          type: 'text',
+          value: getValue('address.region'),
+          error: validationErrorMessages['address.region'],
+        },
+        {
+          name: 'address.postal_code',
+          label: 'Postal Code',
+          type: 'text',
+          value: getValue('address.postal_code'),
+          error: validationErrorMessages['address.postal_code'],
+        },
+        {
+          name: 'address.country',
+          label: 'Country',
+          type: 'text',
+          value: getValue('address.country'),
+          error: validationErrorMessages['address.country'],
+        },
+      ]
+
+    };
+  },
+  forgotPassword : (request, error) => ({
+    title: 'Forgot Password',
+    formAction: `/user/forgot-password?${querystring.stringify(request.query)}`,
+    returnTo: `${request.query.redirect_uri}?status=cancelled`,
+    error: !!error,
+    validationErrorMessages: getValidationMessages(error),
+  }),
+  resetPassword : (request, error) => {
+    const redirectSet = request.query.token != undefined;
+    return {
+      title: title,
+      returnTo: (redirectSet) ? false : `${request.query.redirect_uri}?status=cancelled`,
+      error: !!error,
+      validationErrorMessages: error && error.isBoom ? getValidationMessages(error) : error,
+    };
+  },
+  resetPasswordSuccess : request => {
+    return {
+      title: 'Password Set',
+      linkUrl: `/op/auth?${querystring.stringify({
+        client_id: request.query.client_id,
+        response_type: 'code id_token token',
+        scope: request.query.scope,
+        redirect_uri: request.query.redirect_uri,
+        nonce: 'nonce',
```

Comment:
Is there a reason this is hardcoded to `nonce`? I think this is causing a bug in tathata right now because we expect the randomly-generated nonce we keep in localstorage to be passed back after setting your password.

Reply:
No i'm sure this is a dumb thing that accidentally got left in. I will file an issue for it.